Messaging/StreamingApi.cs

```diff
@@ -35,10 +33,10 @@ public static class StreamingApi
         public static readonly bool IsEnabled = Config.GetBool("send-via-api");
 
         // Client for sending asynchronous requests.
-        private static readonly RestClient Client = new RestClient("http://streaming.quantconnect.com")
+        private static readonly Lazy<RestClient> Client = new Lazy<RestClient>(() => new RestClient("http://streaming.quantconnect.com")
```

Comment:
I'd prefer to delete rather than update this ancient code; which since its a new larger problem suggest we just revert these changes and fix this class properly in a dedicated PR.